pom.xml

```diff
@@ -9,7 +9,7 @@
     <groupId>dev.alangomes</groupId>
     <artifactId>spigot-spring-boot-starter</artifactId>
     <packaging>jar</packaging>
-    <version>0.20.4</version>
+    <version>0.20.5</version>
```

Comment:
Since 0.20.7 is now released, this should be targeted at 0.20.8 now, or maybe 0.21.0, since it is an added feature.